client/src/components/Reviews/Review.jsx

```diff
@@ -1,26 +1,71 @@
-import React from 'react';
+import React, { useState } from 'react';
+import { SingleReviewStars } from '../styles/reviewstyles/SingleReviewStars.styled.js';
+import { ThumbNail } from '../styles/reviewstyles/ThumbNail.styled.js';
+import { Background } from '../styles/reviewstyles/FormBackground.styled.js';
+import { LargePhoto } from '../styles/reviewstyles/LargePhoto.styled.js';
 
-const Review = ({review}) => {
+const Review = ({ review, StarReview }) => {
 
-  // "review_id": 1274559,
-  // "rating": 3,
-  // "summary": "this is kevin's summary",
-  // "recommend": true,
-  // "response": null,
-  // "body": "let''s sogooooo get it together",
-  // "date": "2022-05-26T00:00:00.000Z",
-  // "reviewer_name": "mynickname",
-  // "helpfulness": 3,
-  // "photos": []
+  const [restriction, toggle] = useState(false);
+  const [clicked, controlClicks] = useState(false);
+  const [image, togglePhoto] = useState(false);
+
+  let toggleRestriction = () => {
+    toggle(prevState => !prevState);
+  };
+
+  let oneClick = () => {
+    controlClicks(prevState => !prevState);
+  };
+
+  let clickPhoto = () => {
+    togglePhoto(prevState => !prevState);
+  };
+
+  const enlargeThumbnail = (photo) => {
+    return (
+      <Background onClick={() => clickPhoto()}>
```

Comment:
you can also condense this into one line rather than having clickPhoto being a separate function, by having
`<Background onClick={() => togglePhoto(!image}>`